scripts/generate-releases-manifest.mjs

```diff
@@ -151,7 +151,7 @@ if ( isDevBuild ) {
 	const windowsReleaseInfo = await getWindowsReleaseInfo();
 	releasesData[ 'dev' ][ 'win32' ] = {
 		sha: windowsReleaseInfo.sha1,
-		url: `${ cdnURL }/${ baseName }-win32-v${ version }-${ currentCommit }-full.nupkg`,
+		url: `${ cdnURL }/${ baseName }-win32-${ currentCommit }-full.nupkg`,
```

Comment:
@wojtekn The version is needed for parsing the release information by Squirrel. If we expect dev builds to be updated, we'd need to keep this part.

Reply:
@fluiddot I wasn't sure which was correct - adding the version in both places or removing it - so I removed it to at least point it to the existing file. If we need to add a version, we must first change the `fastlane/Fastfile` to use the version and hash for the dev build file name.

Reply:
> If we need to add a version, we must first change the fastlane/Fastfile to use the version and hash for the dev build file name.

Ah, good point. Yes, we'd need to incorporate the current version (extracted from `package.json`) into the filename.

Reply:
Thanks, added https://github.com/Automattic/dotcom-forge/issues/7439